guides/doc-Administering_Red_Hat_Satellite/topics/Starting_and_Stopping_Satellite.adoc

```diff
@@ -5,7 +5,30 @@
 This is useful when creating a backup of {Project}.
 For more information on creating backups, see xref:backing-up-satellite-server-and-capsule-server[].
 
+ifdef::foreman-el,katello[]
+.Prerequisite
+This section uses the `{foreman-maintain}` command.
+To use the commands in this section, you must install `{foreman-maintain}`, for example:
+
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# yum install `{foreman-maintain}`
+----
+
+Alternatively, you can also use `systemctl` to start and stop services:
```

Comment:
I still struggle with this. With this you still have services running in the background, like dynflow (job processing). I don't really know what the intent of this chapter is. It's really written as all or nothing, but that's really vague. So I don't really have good advice on what to write.

Reply:
I don't remember when this chapter was introduced if I'm honest. I think that we wanted a way to show the foreman-maintain service options. It's technically not a procedure just a reference list of options for starting and stopping. It's totally lacking context. 
I didn't understand your dynaflow comment from the issue, and commented as such. I get it a small bit better. I can add a note in.